### ISSUE TITLE
+ into ,

### ISSUE THREAD

**Issue by kusalstha** [scotch-io/express-sendfile#2]
Hi, I was going through your tutorial and I saw a mistake. 

`res.sendFile(path.join(__dirname + '/index.html'));` shouldn't that be `res.sendFile(path.join(__dirname,  '/index.html'));`

Happy coding!

**Comment by chris-sev** [comment]
Yup you're right. Could you shoot over a pull request with this change?
